### PR TITLE
update questions for ODS file

### PR DESCRIPTION
**Pull request by danielknell**
adds a section to output_brief_response for the ODS file, and updates names of some questions.

### CODE REVIEW

**Comment by TheDoubleK** [comment]
Why are we mixing up the terminology "Requirements" and "Skills and experience"?  Seems to me it would be much better to be consistent and just change the `name` fields everywhere to be "skills and experience" if that's the terminology we're settled on.

**Comment by TheDoubleK** [comment]
Same with "Earliest start date" - that has a slightly different meaning from "Date the specialist can start" - can we really not settle on one thing for those questions?

File: frameworks/digital-outcomes-and-specialists/questions/brief-responses/essentialRequirements.yml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 name: Give evidence of the essential skills and experience
+short_name: Essential skills and experience
```
Comment (TheDoubleK) [inline frameworks/digital-outcomes-and-specialists/questions/brief-responses/essentialRequirements.yml]: With this one can't the `name` be `Essential skills and experience` and keep the long version for `question`?

**Comment by TheDoubleK** [comment]
If these are the only changes required a new key really feels like overkill to me.

File: frameworks/digital-outcomes-and-specialists/questions/brief-responses/essentialRequirements.yml (hunk 1 above)
Comment (danielknell) [inline frameworks/digital-outcomes-and-specialists/questions/brief-responses/essentialRequirements.yml]: this was changed for the new flow. @idavidmcdonald @Wynndow ?

File: frameworks/digital-outcomes-and-specialists/manifests/output_brief_response.yml [hunk 2]
```diff
@@ -7,3 +7,13 @@
     - legacyDayRate
     - niceToHaveRequirements
     - legacyRespondToEmailAddress
+-
+  name: View ODS
```
Comment (idavidmcdonald) [inline frameworks/digital-outcomes-and-specialists/manifests/output_brief_response.yml]: I think we want to have the View ODS section as a separate manifest named 'output_brief_response'.

The View response to requirements section should be moved into a manifest called 'legacy_output_brief_response'.

File: frameworks/digital-outcomes-and-specialists/questions/brief-responses/legacyAvailabilityOutcomes.yml [hunk 3]
```diff
@@ -1,4 +1,5 @@
 name: Date the supplier can start work
+short_name: Earliest start date
```
Comment (idavidmcdonald) [inline frameworks/digital-outcomes-and-specialists/questions/brief-responses/legacyAvailabilityOutcomes.yml]: We shouldn't need to make any changes to the legacy questions as we are not changing anything with the CSV.

File: frameworks/digital-outcomes-and-specialists/questions/brief-responses/legacyAvailabilitySpecialists.yml [hunk 4]
```diff
@@ -1,4 +1,5 @@
 name: Date the specialist can start work
+short_name: Earliest start date
```
Comment (idavidmcdonald) [inline frameworks/digital-outcomes-and-specialists/questions/brief-responses/legacyAvailabilitySpecialists.yml]: We shouldn't need to make any changes to the legacy questions as we are not changing anything with the CSV.

File: frameworks/digital-outcomes-and-specialists/questions/brief-responses/legacyEssentialRequirements.yml [hunk 5]
```diff
@@ -1,4 +1,5 @@
 name: Essential requirements
+short_name: Essential skills and experience
```
Comment (idavidmcdonald) [inline frameworks/digital-outcomes-and-specialists/questions/brief-responses/legacyEssentialRequirements.yml]: We shouldn't need to make any changes to the legacy questions as we are not changing anything with the CSV.

File: frameworks/digital-outcomes-and-specialists-2/questions/brief-responses/niceToHaveRequirements.yml [hunk 6]
```diff
@@ -1,4 +1,5 @@
 name: Nice-to-have requirements
+short_name: Nice-to-have skills and experience
```
Comment (idavidmcdonald) [inline frameworks/digital-outcomes-and-specialists-2/questions/brief-responses/niceToHaveRequirements.yml]: Personally I wouldn't bother making changes to the DOS2 questions as we will copy them all again from DOS 1 at a later point for which the copy is still being changed.

File: frameworks/digital-outcomes-and-specialists/questions/brief-responses/essentialRequirements.yml (hunk 1 above)
Comment (idavidmcdonald) [inline frameworks/digital-outcomes-and-specialists/questions/brief-responses/essentialRequirements.yml]: So I think we are able to change the name. I think we were quite new to the content loader when we started so assumed you needed both a name and question (and therefore set them to be the same thing).

Therefore I think we are able to have `name` be `Essential skills and experience` and not need the new `short_name` key. Sorry!

**Comment by danielknell** [comment]
not updating legacy questions will cause the flows to use different terminology between the summary pagess of old briefs and new.

**Comment by idavidmcdonald** [comment]
I believe this should be OK. The summary pages are going to be relatively different anyway, with different templates and different content (the addition of evidence). 

The current plan that has been discussed is that we are happy to keep the existing page exactly as is.

File: frameworks/digital-outcomes-and-specialists/manifests/output_brief_response.yml (hunk 2 above)
Comment (idavidmcdonald) [inline frameworks/digital-outcomes-and-specialists/manifests/output_brief_response.yml]: I think this comment still stands @danielknell

**Comment by idavidmcdonald** [comment]
Code looks good. Would be nice to rewrite the commit message to be slightly more descriptive though please. 

Version bump also needed.

👍

File: frameworks/digital-outcomes-and-specialists/manifests/legacy_output_brief_response.yml [hunk 7]
```diff
@@ -0,0 +1,9 @@
+-
+  name: View response to requirements
+  questions:
+    - supplierName
+    - legacyAvailabilityOutcomes
+    - legacyAvailabilitySpecialists
+    - legacyDayRate
+    - legacyNiceToHaveRequirements
+    - legacyRespondToEmailAddress
```
Comment (TheDoubleK) [inline frameworks/digital-outcomes-and-specialists/manifests/legacy_output_brief_response.yml]: The "No newline" warnings bug me.  But it doesn't break anything.

**Comment by TheDoubleK** [comment]
Looks fine to me, with or without the final line break :+1:

**Pull request merged**